### PR TITLE
Barrel can now be used as a mix dependency

### PR DESCRIPTION
**Pull request by sivsushruth**

### CODE REVIEW

**Comment by pjhampton** [comment]
This looks cool!! :D 

How can I test? (Sorry to be a pain ;) )

**Comment by sivsushruth** [comment]
Hey check slack @pjhampton . Let me know if you find any bug . Thank you

File: mix.exs [hunk 1]
```diff
@@ -30,41 +30,44 @@ defmodule Barrel.Mixfile do
...
+        :exometer_core,
+        :snappy
       ]
     ]
   end
 
   defp deps do
     [
-      {:lager, "3.0.2"},
+      {:barrel_ex_logger, git: "https://github.com/barrel-db/barrel_ex_logger.git"},
```
Comment (benoitc) [inline mix.exs]: How costly is enough compared to using logger directly? If we use logger, how is it configured?

is this easier to use https://github.com/tolbrino/hut/tree/master/examples/elixir ?

Comment (benoitc) [inline mix.exs]: i meant using hut

**Comment by sivsushruth** [comment]
@benoitc checking . Thanks

**Comment by benoitc** [comment]
thanks!

**Pull request merged**